wallet/src/main/java/com/android/identity_credential/wallet/ui/prompt/passphrase/PassphrasePrompt.kt

```diff
@@ -69,18 +177,130 @@ class PassphrasePrompt(
     ): View =
         ComposeView(requireContext()).apply {
             setContent {
-                IdentityCredentialTheme {
-                    PassphraseEntryField(
-                        constraints = constraints,
-                        checkWeakPassphrase = checkWeakPassphrase,
-                        onChanged = { passphrase, _, donePressed ->
-                            // notify of the typed passphrase when user taps 'Done' on the keyboard
-                            if (donePressed) {
-                                onPassphraseEntered.invoke(passphrase)
-                            }
+                var currPassphrase = ""
+
+                // if fixed length, the IME action button shouldn't do anything because the user
+                // needs to provide exactly the required number of characters; show an arrow icon
+                // rather than a checkmark icon (since with fixed length, the user should not
+                // have the option to "complete" the entry, rather it is completed inherently
+                // when the required number of characters has been entered)
+                val imeAction = if (constraints.isFixedLength()) ImeAction.Go else ImeAction.Done
```

Comment:
Why `ImeAction.Go` and not `ImeAction.None`? (I'm assuming `.None` will not show anything)

Reply:
Right first thought was to use `None` but that doesn't do anything - it resorts to the default case which is worse than the "right arrow" of the "Go" `ImeAction`